pkg/utils/redis_utils_test.go

```diff
@@ -42,7 +44,8 @@ func (s *testUtilsSuite) TestPing(c *C) {
 	c.Assert(err, IsNil)
 }
 
-func (s *testUtilsSuite) TestGetInfo(c C) {
-	_, err := GetRedisInfo(s.addr, "", s.auth)
-	c.Assert(err, IsNil)
+func (s *testUtilsSuite) TestGetInfo(c *C) {
+	// miniredis has no INFO command, we will use qdb instead later.
```

Comment:
miniredis has not INFO command, so we can not test some functions with it.

Later @qiuyesuifeng will replace miniredis with qdb, atm, we can re-add this test.